traffic_control/management/commands/import_traffic_sign_codes.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import os
+
+from django.core.management.base import BaseCommand, CommandError
+from django.test import Client
+from django.urls import reverse
+from django.utils.crypto import get_random_string
+
+from users.models import User
+
+
+class Command(BaseCommand):
+    help = "Import traffic sign code data from a csv file"
+
+    def add_arguments(self, parser):
+        parser.add_argument("filename", help="Path to the traffic sign codes csv file")
+
+    def handle(self, *args, **options):
+        filename = options["filename"]
+        if not os.path.exists(filename):
+            raise CommandError("File {0} does not exist".format(filename))
+
+        with open(filename, mode="r", encoding="utf-8-sig") as csv_file:
```

Comment:
I sort of don't like what's going on here. First of all if we are running this command "locally" aka in the environment that we are running this code, we don't need to use the API client to do the POSTing. At least since receiving these objects are trivial task and there's no special stuff with them. Hmm. Also I don't like this:
`user = User.objects.create_superuser(
                username=username, password=password, email="testadmin@example.com"
            )`
It's just super hacky way to do things... and if we wanted to do it this way aka manage.py command with API client, just read the credentials for the user from environment variables.

Sad to say, but I think this approach is most likely not the way to go - even when the code itself is actually totally fine! Hopefully this does not sound too harsh :)